Gemfile

```diff
@@ -8,3 +11,5 @@ source 'http://rubygems.org'
 gemspec
 
 # gem 'openstudio-standards', '= 0.2.17.rc1', :github => 'NREL/openstudio-standards', :ref => '3.5.0_changes'
+
+gem 'rubocop-performance', '~> 1.20'
```

Comment:
We add it in #204, #206 then, remoove it in #207 then it gets readded here in #210. Wow.